src/Core/Geometry/TriangleMesh.cpp

```diff
@@ -47,52 +26,19 @@ bool TriangleMesh::append( const TriangleMesh& other ) {
     return true;
 }
 
-void TriangleMesh::clearAttributes() {
-    Utils::Attrib<Vector3>::Container v;
-    Utils::Attrib<Vector3>::Container n;
-    std::exchange( v, vertices() );
-    std::exchange( n, normals() );
```

Comment:
isn't the `std::exchange` process designed to be faster?

Reply:
well, vertices() is readOnly.